src/rules/no-tags.js

```diff
@@ -0,0 +1,41 @@
+var _ = require('lodash');
+var rule = 'no-tags';
```

Comment:
do you mind renaming this to 'disallow-tags' or something similar? `no-tags` sounds to me like "no tags are allowed" rather than "this is a list of tags that aren't allowed"